e2e/tests/submissions/job.test.js

```diff
@@ -0,0 +1,111 @@
+const moment = require('moment');
+const { ResponseInterceptor } = require('../../utils/interceptors');
+const { login } = require('../../utils/user');
+const { FormSubmitter } = require('../../utils/form');
+const routes = require('../../utils/routes');
+
+describe('job submissions', () => {
+  let context;
+  let page;
+
+  beforeEach(async () => {
+    context = await browser.createIncognitoBrowserContext();
+    await login(context);
+
+    page = await context.newPage();
+    await page.goto(routes.JOB_SUBMISSION);
+  });
+
+  it('submits and updates a job', async () => {
+    const interceptor = new ResponseInterceptor(page);
+    const formSubmitter = new FormSubmitter(page);
+    await formSubmitter.submit({
+      title: 'Software developer',
+      external_job_identifier: '07587',
+      institutions: [{ value: 'CERN' }, { value: 'Berkley' }],
+      regions: ['Europe', 'Asia'],
+      field_of_interest: ['cond-mat', 'astro-ph'],
+      ranks: ['POSTDOC', 'MASTER'],
+      experiments: [{ legacy_name: 'Atlas' }, { legacy_name: 'CMS' }],
+      url: 'https://someinfo.com',
+      deadline_date: moment()
+        .add(1, 'day')
+        .format('MMMM DD, YYYY'),
```

Comment:
let's move this logic to `fillDateField` and keep data clean, in its original format. Since we need this logic only to be able to select dom elements.